### PR TITLE
RIA-3541 Ensure consistency for ia-ccd-e2e-tests

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###

[RIA-3541](https://tools.hmcts.net/jira/browse/RIA-3541)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/features/ftpa-display-final-decision-scenario-1.feature [hunk 1]
```diff
@@ -25,13 +24,14 @@ Feature: Final display for FTPA decision and reasons
     And I switch to be a `Case Officer`
     And I create case summary
     And I generate the hearing bundle
+    And I wait for 30 seconds
```
Comment (divisathyan) [inline e2e/features/ftpa-display-final-decision-scenario-1.feature:27]: Is this accepted? The E2E tests will get delayed

**Review by divisathyan — COMMENTED** [review]
Please look into the comment

File: e2e/features/ftpa-display-final-decision-scenario-1.feature (hunk 1 above)
Comment (divisathyan) [reply e2e/features/ftpa-display-final-decision-scenario-1.feature:27]: @burale checked and told it is acceptable

**Review by divisathyan — APPROVED** [review]
LGTM

**Review by divisathyan — APPROVED** [review]
LGTM

**Comment by rohith-tiru** [comment]
Results file attached
[ria_2919_scenario1.txt](https://github.com/hmcts/ia-ccd-e2e-tests/files/5378245/ria_2919_scenario1.txt)

**Review by divisathyan — APPROVED** [review]
LGTM

**Pull request merged**